rocAL/rocAL_pybind/amd/rocal/decoders.py

```diff
@@ -107,7 +107,7 @@ def image(*inputs, user_feature_key_map=None, path='', file_root='', annotations
             "max_width": max_decoded_width,
             "max_height": max_decoded_height,
             "dec_type": decoder_type}
-        decoded_image = b.ImageDecoderShard(Pipeline._current_pipeline._handle, *(kwargs_pybind.values()))
+        decoded_image = b.imageDecoderShard(Pipeline._current_pipeline._handle, *(kwargs_pybind.values()))
```

Comment:
you need to make sure all the tests on python are changed accordingly because of the name change in api

Reply:
We have changed all the API names accordingly. For readers.py and decoders.py, we only changed the API names for the file reader and image decoder. Other readers and decoders are changed in the next metanodes PR and the example files are changed in the python examples PR